.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,37 @@
+## Description
+
+TODO: Link to a JIRA ticket if one has been created for this feature/issue.
+Use the following format for a link: `[FEDX-100](https://openedx.atlassian.net/browse/FEDX-100)`
+
+TODO: Include a detailed description of the changes in the body of the PR.
+
+TODO: If relevant, provide a URL to updated Pattern Library documentation.
+
+## Manual Test Plan
+
+TODO: If your changes require manual tests, include instructions for any required manual tests, and any manual testing that has already been performed.
+
+## Testing Checklist
+- [ ] Write unit tests for all new features.
+- [ ] Write accessibility (a11y) tests for all new UI.
+- [ ] Manually test right-to-left languages and i18n
```

Comment:
I'm not sure this is possible since the UI Toolkit doesn't support i18n as yet. It seems a shame to remove it since we clearly do want developers thinking about this, but maybe we have to for now.